tests/unit/tooltip/common.js

```diff
@@ -19,6 +19,7 @@ common.testWidget( "tooltip", {
 		},
 		show: true,
 		track: false,
+		textDir: null,
```

Comment:
These options are alphabetized. All files will need to be updated for the correct order.